.github/workflows/docker.yaml

```diff
@@ -40,8 +40,8 @@ jobs:
             VERSION=${{ steps.meta.outputs.tags }}
             PROJECT=..
           file: docker/goDockerfile
-          push: true
-          tags: ${{ steps.meta.outputs.tags }}
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: "${{ steps.meta.outputs.tags }},ghcr.io/glothriel/wormhole-controller:latest"
```

Comment:
Remember that `on: pull_request` is triggered only when creating a pull request - later on it will be simple `push`. So I assume that with current setup this will only be triggered while pushing specific tag. 

And remember that `test-tag` will also overwrite `latest` if it will be pushed but it shoudn't be a problem if helm charts will use fixed versions.

Reply:
> Remember that on: pull_request is triggered only when creating a pull request - later on it will be simple push. So I assume that with current setup this will only be triggered while pushing specific tag.

I don't think so, just checked that

Reply:
> And remember that test-tag will also overwrite latest if it will be pushed but it shoudn't be a problem if helm charts will use fixed versions.

I'm aware, but i didn't want to spend 3 days to handle this now.